core/src/player.rs

```diff
@@ -356,6 +357,9 @@ pub struct Player {
     /// Any compatibility rules to apply for this movie.
     compatibility_rules: CompatibilityRules,
 
+    /// A map from gamepad buttons to key codes.
+    gamepad_button_mapping: HashMap<GamepadButton, KeyCode>,
```

Comment:
Not a major issue or anything blocking ;D Just noticed that since the key is an enum, this could also be Vec<Option> or a perfect hash.
Not required to change, just an observation.